GapBackfill/Helpers/SqlHelpers.cs

```diff
@@ -136,6 +144,12 @@ internal static async Task RecordDeviceBackfillRequest(Types.BackfillRequest req
                 {
                     await connection.OpenAsync();
 
+                    using (SqlCommand command = new SqlCommand(SqlHelpers.SQL_CREATE_TABLE_BACKFILL_REQUEST, connection))
+                    {
+                        if (await command.ExecuteNonQueryAsync() != 1)
+                            throw new Exception($"Cannot create table for backfill request in SQL:{JsonConvert.SerializeObject(request)}");
```

Comment:
Should we throw something more specific than a general exception?

Reply:
Maybe...but I think for production readiness the error handling needs to improved anyways...